stock_move_backdating/stock.py

```diff
@@ -81,10 +81,10 @@ def on_change_date_backdating(self, cr, uid, ids, date_backdating,
         if (dt > NOW):
             warning = {'title': _('Error!'), 'message': _(
                 'You can not process an actual movement date in the future.')}
-            values = {
-                'date_backdating': NOW.strftime(DEFAULT_SERVER_DATETIME_FORMAT)
-            }

```

Comment:
@llacroix : It doesn't break the functionality. The bug for multiple popup message is only for date field. After popup message you change the date if you want.